index.hbs

```diff
@@ -76,11 +76,85 @@ into the {body} of the default.hbs template --}}
+                  </p>
+                </div>
+
+                <h3 class="ieee-blue">What We Do</h1>
+                <div class="paragraph-div">
+                  <p class="paragraph">
+                    We aim to promote professional development, encourage academic growth, and organize social activities within the greater engineering community. <br />
+                    Some of the activities we host:
+                    <div class="row">
+                      <div class="two-columns">
+                        <ul class="paragraph">
+                          <li>Technical Workshops</li>
+                          <li>Resume and Interview Workshops</li>
+                          <li>Startup Fair</li>
+                          <li>Dinner with Corporate Sponsors</li>
+                        </ul>
+                      </div>
+                      <div class="two-columns">
+                        <ul class="paragraph">
+                          <li>Company Info-sessions</li>
+                          <li>Mentorship Program</li>
+                          <li>Two Decals (Hope, Micromouse)</li>
+                          <li>Socials (Dinner, bowling, ice-skating, etc.)</li>
+                        </ul>
+                      </div>
+                    </div>
+                  </p>
+                </div>  {{!-- End of paragraph-div --}}
+
+
+                <div class="stats-div text-center div-spacing">
+                  <div class="row">
+                    <div class="three-columns stats-columns">
+                      <div class="stats-bubbles">
+                        <span class="stats">500 General Members</span>
+                      </div>
+                    </div>
+                    <div class="three-columns stats-columns">
+                      <div class="stats-bubbles">
+                        <div class="stats">50 Officers and Directors</div>
+                      </div>
+                    </div>
+                    <div class="three-columns stats-columns">
+                      <div class="stats-bubbles">
+                        <span class="stats">100% Acceptance</span>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <!--<div class="stats-div div-spacing">
+                  <span class="stats-bubbles"><span class="stats">500 General Members</span></span><span class="stats-bubbles"><span class="stats">50 Club Officers and Directors</span></span><span class="stats-bubbles"><span class="stats">100% General Member Acceptance</span></span>
+                </div>-->
+
+                <div class="join-div div-spacing">
+                  <h2 class="ieee-blue">Join Our Community!</h4>
+                  <a href="/apply" type="button" class="ieee-btn link"><span class="btn-text"> Apply Now</span></a>
```

Comment:
There should be a hover selector so that the box is noticeable when you hover over it.